Doc - adds info about user to be used to start the starter (#209)
arangodb-helper/arangodb: docs/Manual/Deployment/ActiveFailover/UsingTheStarter.md

Using the ArangoDB Starter
==========================

This section describes how to start an _Active Failover_ setup using the tool
[_Starter_](../../Programs/Starter/README.md) (the _arangodb_ executable).

{% hint 'warning' %}
It is **strongly suggested** to **not** run the _ArangoDB Starter_ process

> (_arangodb_) from a system _user_ with full privileges, e.g. _root_ under Linux.

Reviewer: I don't see why "user" should be italic
```suggestion
(_arangodb_) as system user with full privileges, e.g. _root_ under Linux.
```

> We suggest to create a dedicated _user_ with limited privileges for this purpose.

Reviewer: ```suggestion
We suggest to create a dedicated system user with limited privileges for this purpose.
```

Please refer to the [Linux user used to run _arangodb_](../ArangoDBStarter/README.md#linux-user-used-to-run-arangodb)

> section of the Starter Deployment page for further details.

Reviewer: Can we add some info for macOS and Windows as well?

Author: Simran - can you?

Reviewer: On Windows in a Command Prompt or PowerShell you can do `runas /user:arangodb "arangodb args"` but I couldn't figure out how to escape certain characters. Is our general recommendation to use a restricted system user account on all platforms? I know that MySQL creates a separate (hidden?) account, which has different file & folder permissions for instance. I don't know what type of account it actually is however.

Author: please can you open a ticket for this? we incorporate it in a later PR - thanks

Run the above command on machine A, B & C.

> An Agent and a Single Instance will be started on each machine. One of the Single Instance will 

Reviewer: One of the Single Instance<ins>s</ins> (plural) I think?

have the _leader_ role, the others two the _follower_ role.  

> If you want to have only one follower, add (only valid while bootstrapping),  

Reviewer: only **possible** while bootstrapping ? Maybe move this remark to the end of the sentence?
